JT/01_environment_setup/README.adoc

```diff
@@ -0,0 +1,406 @@
+
+. Using WinSCP
+If you chose to launch WinSCP, the program should start automatically after installation.
+You can now use WinSCP to connect to remote servers using protocols like FTP, SFTP, or SCP. You'll need to provide the server details, such as hostname, username, and password, to establish a connection.
+
+=== [[oracle-sql]]Oracle SQL
+
+Oracle SQL Developer is a free graphical tool that enhances productivity and simplifies database development tasks. With SQL Developer, you can browse database objects, run SQL statements and SQL scripts, edit and debug PL/SQL statements, manipulate and export data, and view and create reports. You can connect to Oracle databases, and you can connect to selected third-party (non-Oracle) databases, view metadata and data, and migrate these databases to Oracle.
+
+Any project as a whole is closely related to DB's and to properly test threads it is necessary to be able to check values in DB. Here is a guide to configuring connections with Oracle SQL Developer.
+
+. Browse to this link https://www.oracle.com/database/sqldeveloper/technologies/download/.
++
+image::resources/oracle-01.png[]
+
+. Click on download.
++
+image::resources/oracle-02.png[]
+
+. Login in Oracle account or create a new account. It is free
++
+image::resources/oracle-03.png[]
+
+. Extract the downloaded zip file.
++
+image::resources/oracle-04.png[]
+
+. Open your computer C drive & program files. Cut & Paste the extracted file there.
++
+image::resources/oracle-05.png[]
+
+. Open the folder. Then open sqldeveloper folder.
++
+image::resources/oracle-06.png[]
+
+. Find the sqldeveloper exe file there. Create a shortcut on desktop.
++
+image::resources/oracle-07.png[]
+
+. Open the shortcut. It will give you welcome page. After installation you should see this picture:
++
+image::resources/oracle-08.png[]
+
+Hence, your installation is successful.
+Or as an alternative, you can add it to IntelliJ IDEA.
+
+=== [[putty]]PuTTY
+
+PuTTY is a free, open-source terminal emulator, serial console, and network file transfer application. It supports various network protocols, including SSH, Telnet, and SCP, and is commonly used for remote access to servers and network devices.
+
+=== Download PuTTY
++
+Open your web browser and go to the PuTTY download page: https://www.putty.org/
+
+Click on the link that says "You can download PuTTY here" to go to the download section.
+
+Under the "Package files" section, download the `putty-<version>-installer.msi` file for Windows. The `<version>` part will be replaced by the current version number of PuTTY.
+
+image::resources/putty-01.png[]
+
+. Run the Installer
++
+Locate the downloaded MSI file (usually in your "Downloads" folder).
+
+Double-click on the `putty-<version>-installer.msi` file to start the installation process.
+
+=== Installation Wizard
++
+**Welcome Screen**: Click "Next" to continue.
++
+image::resources/putty-02.png[]
```

Comment:
Please check if screenshot is available in .adoc version

Reply:
it was fixed.